CRM/Core/I18n/SchemaStructure.php

```diff
@@ -402,13 +402,14 @@ public static function &widgets() {
         ],
         'civicrm_option_value' => [
           'label' => [
-            'type' => "Text",
```

Comment:
Why did GenCode remove this???? Seems to have caused test fails & also seems just wrong

Reply:
OK - I got past this by adding `type` to the xml for the field (rather than it being implicit) - I didn't fix the underlying gencode bug....